### PR TITLE
Updated patch file to build base-notebook Dockerfile on ppc64le

### PR DESCRIPTION
**Pull request by seth-priya**
This PR is to fix the following issue
https://github.com/jupyter/docker-stacks/issues/377

This updates the ppc64le patch file and make build-all and make test-all succeeds post this update.

### CODE REVIEW

File: base-notebook/Dockerfile.ppc64le.patch [hunk 1]
```diff
@@ -62,30 +63,33 @@
...
 -RUN conda install --quiet --yes \
--    'notebook=4.2*' \
--    jupyterhub=0.7 \
+-    'notebook=5.0.*' \
+-    'jupyterhub=0.7.*' \
+-    'jupyterlab=0.18.*' \
 -    && conda clean -tipsy
++# Temporary workaround for https://github.com/jupyter/docker-stacks/issues/210
++# Stick with jpeg 8 to avoid problems with R packages
++RUN echo "jpeg 8*" >> /opt/conda/conda-meta/pinned
```
Comment (parente) [inline base-notebook/Dockerfile.ppc64le.patch]: This is no longer in the original. Does it need to be for PPC?

File: base-notebook/Dockerfile.ppc64le.patch [hunk 2]
```diff
@@ -62,30 +63,33 @@
...
++
++# Install Jupyter notebook as jovyan
 +RUN yes | pip install --upgrade pip
 +RUN yes | pip install --quiet --no-cache-dir \
 +    'notebook==4.2.*' \
```
Comment (parente) [inline base-notebook/Dockerfile.ppc64le.patch]: This is backlevel now compared to x86. 5.0 is the latest.

**Comment by parente** [comment]
Reading diffs of diffs makes my head hurt, but I left a couple of comments. Hopefully they make sense.

**Comment by seth-priya** [comment]
@parente, thanks for the review. I will work on these and get back with the required changes.

**Comment by seth-priya** [comment]
@parente I have updated Dockerfile.ppc64le and re-generated the patch file to address your review comments. Request you to please let me know your suggestions. Thanks!

**Comment by parente** [comment]
LGTM. Thanks. Merging!

**Comment by seth-priya** [comment]
thanks!

**Pull request merged**